src/plugins/inputs/setpass.js

```diff
@@ -0,0 +1,41 @@
+var bcrypt = require("bcrypt-nodejs");
+var Msg = require("../../models/msg");
+
+module.exports = function(network, chan, cmd, args) {
+	if (cmd !== "setpass") {
```

Comment:
Do we really need a command to change password? It might also be confused for a server command.